src/components/Header.jsx

```diff
@@ -0,0 +1,29 @@
+import { Link } from 'react-router-dom';
+import covid from '../img/covid.svg';
+
+const Header = () => (
+  <nav className="navbar bg-blue f-gill fc-white">
+    <div className="container-fluid d-flex justify-content-around">
+      <Link to="/">
+        <box-icon name="left-arrow" animation="burst-hover" color="#ffffff"></box-icon>
+      </Link>
```

Comment:
- [ ] This button should route back to the previous page, not to the `/` route. Please update it.

Reply:
Thanks, the navigation is updated.